prophesee_ros_driver/launch/prophesee_publisher.launch

```diff
@@ -7,13 +7,20 @@
     <param name="publish_imu" value="true" />
 
     <!-- Path to the file with the camera settings -->
-    <param name="bias_file" value="" />
+    <!--param name="bias_file" value="$(find prophesee_ros_driver)/cfg/gen3_low_sensitive_settings.bias" /-->
 
-    <!-- Maximum event rate in kEv/s -->
+    <!-- Maximum event rate in kEv/s. With 0 does not set any limit-->
     <param name="max_event_rate" value="6000" />
 
+    <!-- Streaming rate in Hz for the events array. By default is 64 microsendos or max 100 events in array -->
+    <param name="event_streaming_rate" value="30" />
```

Comment:
Could you please comment this line?

Reply:
I could set it to 0 as a default value. It will have the same effect. Do you agree with this?

Reply:
Thanks for changing to 0.
I I understand it right, the effect won't be the same, because it'll take the default value that is different from 30. 
From our side, we don't want to limit event rate to any rate. 
Why do you want to limit event rate ?